references/oidc-mock/pipeline.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# deploy Apigee API Proxy
+mvn install -P"$APIGEE_ENV" -Dapigee.config.options=update
+
+# execute integration tests
+npm i
```

Comment:
we can write `npm install` instead of `npm i` for those that don't know the alias